charts/cloudzero-agent/README.md

```diff
@@ -75,6 +83,9 @@ insightsController:
     enabled: false
     patterns:
       - '.*' # -- match all annotations. This is not recommended.
+cert-manager:
+  # -- Your cluster may already have cert-manager running, in which case this value can be set to false.
+  enabled: true
```

Comment:
What will happen if a cluster manages to get 2 cert-managers installed? I think it's safe that we're defaulting to true here, but concerned if folks don't read the docs and they install our cert-manager over their existing cert manager?

Reply:
there shouldn't actually be any issue. I've run with multiple cert-managers in my test cluster.  but, one of our beta customers did report that they had to disable the one deployed by us because of issues with their previously deployed cert-manager. so, I think it should be avoided if possible

so, we could set this to null and have a schema.json require `cert-manager.enabled` to be set explicitly, or have a helm template fail if it's not supplied. does that sounds like a good idea?

Reply:
All good - let's leave it as is.